pkg/addon/addon.go

```diff
@@ -844,6 +844,18 @@ func RenderArgsSecret(addon *InstallPackage, args map[string]interface{}) *unstr
 	return u
 }
 
+// deleteAddonSecret delete the addon's parameter secret
+func deleteAddonSecret(ctx context.Context, k8sClient client.Client, addonName string) {
+	var sec v1.Secret
+	err := k8sClient.Get(ctx, client.ObjectKey{Namespace: types.DefaultKubeVelaNS, Name: addonutil.Addon2SecName(addonName)}, &sec)
+	if err == nil {
+		err = k8sClient.Delete(ctx, &sec)
+		if err != nil {
+			fmt.Printf("fail to delete %v addon parameter file", addonName)
+		}
```

Comment:
```
if err := k8sClient.Get(ctx, client.ObjectKey{Namespace: types.DefaultKubeVelaNS, Name: addonutil.Addon2SecName(addonName)}, &sec); err == nil {
		if err = k8sClient.Delete(ctx, &sec); err != nil {
			fmt.Printf("fail to delete %v addon parameter file", addonName)
		}
......
}
```

We also should consider the situation of if error is `errNotFound`

Reply:
this code is only for e2e test. the real fix is: https://github.com/kubevela/kubevela/pull/6065(after it was merged, this code will be removed)